src/Playwright/API/Generated/Options/BrowserNewContextOptions.cs

```diff
@@ -47,6 +47,7 @@ public BrowserNewContextOptions(BrowserNewContextOptions clone)
         {
             if (clone == null) return;
             AcceptDownloads = clone.AcceptDownloads;
+            BaseURL = clone.BaseURL;
```

Comment:
why got this line changed?

Reply:
It's possible the change wasn't included in the roll before, for one reason or another. 

I *really* want to merge the bots. Hopefully that can happen today.

Reply:
But the tests were passing before right?

Reply:
But of course. :)